Patterns/Japanese/Japanese-Numbers.yaml

```diff
@@ -81,43 +77,60 @@ RoundDirectListJpn: !list
     - 万
     - 億
     - 兆
-DigitalNumberRegex: !simpleRegex
-  def: ((?<=(\d|\b))(k|t|m|g)(?=\b))
+ZeroToNineIntegerRegexJpn: !simpleRegex
+  def: '[一二三四五六七八九]'
 ZeroToNineJpnFullHalfRegexJpn: !simpleRegex
   def: '[\d１２３４５６７８９０]'
+ZeroToNineJpnFullRegexJpn: !simpleRegex
+  def: '[\d1234567890]'
 DigitNumRegex: !nestedRegex
   def: '{ZeroToNineJpnFullHalfRegexJpn}+'
   references: [ZeroToNineJpnFullHalfRegexJpn]
-PercentageRegex: !simpleRegex
-  def: .+(?=パ\s*ー\s*セ\s*ン\s*ト)|.*(?=[％%])
-DoubleAndRoundJpnRegex: !nestedRegex
-  def: '{ZeroToNineJpnFullHalfRegexJpn}+(\.{ZeroToNineJpnFullHalfRegexJpn}+)?\s*[万億]{1,2}(\s*(以上))?'
+RoundNumberIntegerRegexJpn: !simpleRegex
+  def: '[十百千万億兆]'
+AllIntRegexJpn: !nestedRegex
+  def: (({ZeroToNineIntegerRegexJpn}|{ZeroToNineJpnFullHalfRegexJpn}|[十])\s*{RoundNumberIntegerRegexJpn}){1,2}\s*(\s*[以上]?)
+  references: [ZeroToNineIntegerRegexJpn, ZeroToNineJpnFullHalfRegexJpn, RoundNumberIntegerRegexJpn]
+OrdinalRegexJpn: !nestedRegex
+  def: (だい|第){AllIntRegexJpn}
+  references: [AllIntRegexJpn]
+OrdinalNumbersRegex: !nestedRegex
+  def: (だい|第){ZeroToNineJpnFullHalfRegexJpn}+
   references: [ZeroToNineJpnFullHalfRegexJpn]
-FracSplitRegex: !simpleRegex
-  def: 分\s*の
-ZeroToNineIntegerRegexJpn: !simpleRegex
-  def: '[一二三四五六七八九]'
 NegativeNumberTermsRegexJpn: !simpleRegex
   def: '(マイナス)'
 NegativeNumberTermsRegexNum: !simpleRegex
   def: (?<![-－])[-－]
 NegativeNumberSignRegex: !nestedRegex
   def: ^{NegativeNumberTermsRegexJpn}.*|^{NegativeNumberTermsRegexNum}.*
   references: [NegativeNumberTermsRegexJpn, NegativeNumberTermsRegexNum]
-#IntegerExtractor
-RoundNumberIntegerRegexJpn: !simpleRegex
```

Comment:
Why so many regex deleted?
Please try to keep the YAML file as similar as possible between languages.